content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,178 @@
+
+{{< text bash >}}
+$ kubectl -n istio-system edit gateway
+{{< /text >}}
+
+...and modify the `tls` section corresponding to the `https-default` port as follows:
+
+{{< text bash >}}
+$ kubectl -n istio-system \
+  patch gateway istio-autogenerated-k8s-ingress --type=json \
+  -p='[{"op": "replace", "path": "/spec/servers/1/tls", "value": {"credentialName": "ingress-cert-staging", "mode": "SIMPLE", "privateKey": "sds", "serverCertificate": "sds"}}]'
+{{< /text >}}
+
+Now it's time to setup a demo application.
+
+## Setting up a demo application
+
+We will be using a simple `helloworld` application for this tutorial. The following command will spin up the `Deployment` and `Service` for the demo application and expose the service using an `Ingress` resource that will be handled by `istio-ingressgateway`.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: helloworld
+  labels:
+    app: helloworld
+spec:
+  ports:
+  - port: 5000
+    name: http
+  selector:
+    app: helloworld
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: helloworld
+spec:
+  template:
+    metadata:
+      labels:
+        app: helloworld
+    spec:
+      containers:
+      - name: helloworld
+        image: istio/examples-helloworld-v1
+        resources:
+          requests:
+            cpu: "100m"
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 5000
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: istio
+  name: helloworld-ingress
+spec:
+  rules:
+    - host: "$INGRESS_DOMAIN"
+      http:
+        paths:
+          - path: /hello
+            backend:
+              serviceName: helloworld
+              servicePort: 5000
+---
+EOF
+{{< /text >}}
+
+{{< tip >}}
+Notice the use of `INGRESS_DOMAIN` variable we defined earlier
+{{< /tip >}}
+
+Now we should be able to access our demo application via HTTP:
+
+{{< text bash >}}
+$ curl http://$INGRESS_DOMAIN/hello
+Hello version: v1, instance: helloworld-5d498979b6-jp2mf
+{{< /text >}}
+
+HTTPS access still won't work as we don't have any TLS certificates. Let's fix that.
+
+## Getting a "Let's Encrypt" certificate issued using the "cert-manager"
+
+At this point our Istio installation should have "cert-manager" up and running with two `ClusterIssuer` resources configured (for production and staging ACME-endpoints provided by "Let's Encrypt"). We will be using staging endpoint for this tutorial (feel free to try swapping `letsencrypt-staging` for `letsencrypt` to get a browser-trusted certificate issued).
+
+In order to have a certificate issued and managed by the "cert-manager" we need to create a `Certificate` resource:
```

Comment:
No double-quotes around cert-manager. It's a proper name, so no "the" in front of it either. Please review other uses in the doc accordingly.